src/api/local-storage-provider.model.ts

```diff
@@ -65,16 +65,29 @@ export class LocalStorageProvider extends PersistenceStoreProvider {
     monthDataModel: MonthDataModel
   ): Promise<void> {
     const revisionKey = monthDataModel.scheduleKey.getRevisionKey(revisionType);
-    let revision = "";
+    let revision;
     try {
       const document = await this.storage.get(revisionKey);
       revision = document._rev;
-    } catch {}
-    this.storage.put({
-      _rev: revision,
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      error.status !== 404 && console.error(error);
```

Comment:
Wyłączyłem informowanie o błędzie dla 404 - missing document. Takie jest flow, że jeśli nie znajdzie dokumentu to rzuca error

Reply:
Nie wiedzę gdzie wyrzucasz ten wyjątek. 
Na razie po prostu go nie pokazujesz